kolibri/plugins/learn/assets/src/views/assessment-wrapper/index.vue

```diff
@@ -117,15 +117,15 @@ oriented data synchronization.
       shake: false,
       firstAttempt: true,
       complete: false,
-      correct: true,
+      correct: 0,
       itemError: false,
     }),
     methods: {
       updateAttemptLogMasteryLog({
         correct,
         complete,
         firstAttempt = false,
-        hinted = false,
+        hinted,
         answerState,
         simpleAnswer,
       }) {
```

Comment:
Interesting way to filter out the properties of an object. Is there a reason to do it this way rather than creating separate parameters for each property (besides making the function call nicer)?